lib/screens/language_domain/language_game_three.dart

```diff
@@ -0,0 +1,267 @@
+import 'dart:convert';
+import 'dart:io';
+
+import 'package:http/http.dart' as http;
+import 'package:flutter/material.dart';
+import 'dart:async';
+import 'dart:math';
+import 'enum.dart';
+
+class LanguageGameThree extends StatefulWidget {
+  const LanguageGameThree({super.key});
+
+  @override
+  State<LanguageGameThree> createState() => _LanguageGameThreeState();
+}
+
+class _LanguageGameThreeState extends State<LanguageGameThree> {
+  final String baseUrl = Platform.isAndroid
```

Comment:
convert to a a contants variable for URL in folder constants

Reply:
Done